WordPress/src/main/java/org/wordpress/android/ui/prefs/notifications/NotificationsSettingsFragment.java

```diff
@@ -567,7 +578,7 @@ private void configureBlogsSettings(PreferenceCategory blogsCategory, boolean sh
     }
 
     private void configureFollowedBlogsSettings(PreferenceCategory blogsCategory, final boolean showAll) {
-        if (!isAdded()) {
+        if (!isAdded() || blogsCategory == null) {
```

Comment:
1) Suggestion (💡): It wasn't obvious to me why this change happened and as such I actually played a bit with it to understand more about it. Ideally, I would have either made this change a separate commit or I would have added a commit description on why this extra bit of logic was added here (ie. null pointer exception because of this and that).
2) Suggestion (💡): I would consider adding `@Nullable` annotations everywhere related to this change to make it more obvious what is what. For example, I would have added the `@Nullable` annotation on the `blogsCategory` parameter, just so that it becomes more evident that the `blogsCategory.removeAll();` line might produce null pointer exception. Then, I would have also considered to add the `@Nullable` annotation on the `mFollowedBlogsCategory` field to indicate that this can be nullable going forward, just to avoid any future surprises with it. Wdyt?

Reply:
Great suggestions! 

@Nullable annotation added in 5fa620e.